remoter-client/src/wrapper.r

```diff
@@ -1,16 +1,32 @@
 require("RJSONIO")
 require("paws")
 require("zeallot")
+require("ids")
 
 load_config <- function() {
     config <- list(
-        cluster_env = Sys.getenv("CLUSTER_ENV", "staging"),
+        cluster_env = Sys.getenv("CLUSTER_ENV", "development"),
         sandbox_id = Sys.getenv("SANDBOX_ID", "default"),
         aws_account_id = Sys.getenv("AWS_ACCOUNT_ID", "242905224710"),
         aws_region = Sys.getenv("AWS_DEFAULT_REGION", "eu-west-1")
     )
 
+    config[["aws_config"]] <- list(
+        region = config$aws_region
+    )
+
+    if(config$cluster_env == 'development') {
+        config$aws_config[['endpoint']] <- 'http://host.docker.internal:4566'
+        config$aws_account_id <- '000000000000'
+
+        assignInNamespace("update_endpoint_for_s3_config",function(request) {
+            return(request)
+        },ns="paws.common")
```

Comment:
This fixes an issue in `paws` where for some reason it tries to connect to a subdomain (`biomage-results-staging.host.docker.internal`) instead of just the endpoint like every other service. This mock disables the function that makes it behave weirdly in development.

Reply:
maybe add a code comment about it, because in the future we may forget why this line of code is there and delete it

Reply:
Done